### ISSUE TITLE
Microsoft Learning Pathways Errors in PROD

### ISSUE THREAD

**Issue by shawngr** [pnp/custom-learning-office-365#460]
>**WEB PART NOT LOADING**: If you are getting the error: "Microsoft 365 learning pathways has a configuration issue. Ask your administrator for assistance". Please review the [Diagnosing Pre-Check](https://github.com/pnp/custom-learning-office-365/blob/main/installation/DiagnosisPreCheck.md) guidance before submitting your bug.

## Submit your question
My customer receives the pink error messages consistently on the following pages when using MLP in Chrome (and they believe others experience it in Edge):

-	Learning Pathways administration link
-	Microsoft 365 training link
-	Teams Digital Workplace Community link
-	Welcome to Microsoft Teams link

Attached are screenshots in the case the specific error message is helpful in investigating the root cause and resolving. The customer  worries about how many users are running into the same scenario. They tried the following troubleshooting steps that haven’t resolved the issue:
-	Open new browser session in Chrome
-	Clear cache in Chrome and open new session

![Learning Pathways administration errors](https://user-images.githubusercontent.com/65291859/108305543-32f71580-7178-11eb-9163-c2f4beb86398.JPG)
![Microsoft 365 training MLP errors](https://user-images.githubusercontent.com/65291859/108305545-338fac00-7178-11eb-80be-fedeb82aac34.JPG)
[socialsecuritygov.sharepoint.com-1607007526954.log](https://github.com/pnp/custom-learning-office-365/files/6000264/socialsecuritygov.sharepoint.com-1607007526954.log)
![Teams Digital Workplace Community MLP errors](https://user-images.githubusercontent.com/65291859/108305547-338fac00-7178-11eb-8aa7-6629a288f39f.JPG)
![Welcome to Microsoft Teams MLP Errors](https://user-images.githubusercontent.com/65291859/108305549-34284280-7178-11eb-9698-a8b94bc6ce9e.JPG)

### What is the context of your question

- [ ] Installation
- [x ] Learning Pathways Content
- [ ] SharePoint Success Content
- [ ] Administration
- [ ] Custom Content
- [ ] Viewer Web Part

### What is your question
My Ask 😊 Can the customer speak to someone about their issues and  the best way to implement Learning Pathways, rather than stumbling into issues and limitation by trail-and-error?

>Provide related screen-shots to help us understand your question

**Comment by shawngr** [comment]
[Error_Displayed_Default_level_socialsecuritygov.sharepoint.com-1613669539505.log](https://github.com/pnp/custom-learning-office-365/files/6005386/Error_Displayed_Default_level_socialsecuritygov.sharepoint.com-1613669539505.log)
[Error_Displayed-Errors_Only-socialsecuritygov.sharepoint.com-1613669496661.log](https://github.com/pnp/custom-learning-office-365/files/6005387/Error_Displayed-Errors_Only-socialsecuritygov.sharepoint.com-1613669496661.log)
![screenshot fo metadata jon for log 1613666599248 or 1613666638734](https://user-images.githubusercontent.com/65291859/108410954-0d105600-71f6-11eb-9a88-a075d8d4c9ad.jpg)
[socialsecuritygov.sharepoint.com-1613666599248.log](https://github.com/pnp/custom-learning-office-365/files/6005388/socialsecuritygov.sharepoint.com-1613666599248.log)
[socialsecuritygov.sharepoint.com-1613666638734_Errors_only.log](https://github.com/pnp/custom-learning-office-365/files/6005389/socialsecuritygov.sharepoint.com-1613666638734_Errors_only.log)

**Comment by FormerlyJelly** [comment]
Hello shawngr!

How is the Visitors group of the Communications site configured? This issue can be caused by security configuration, and is the first thing that we should check. Is this tenant a standard O365 tenant, or is it Government specific? The security between these tenants may differ and may contribute to the issue you are experiencing. At this time Github support is the best avenue available for issues with Learning Pathways.

**Comment by shawngr** [comment]
Johnathan,

I'm a Premier Support account manager who opened this case in behalf of the customer.  I do not have access their environment.
I'm the go-between.

Is it okay if I had the customer to this message?

Thank you,
Shawn Grant |Customer Success Account Manager |Federal
Healthcare and Benefits | Microsoft Corporation
W: (410) 564-2720 | C: (443) 676-8337 | shawngr@microsoft.com<mailto:shawngr@microsoft.com>
US National Support_ 1.800.936.3200 | Non-US National Support_1.800.936.3100

**Comment by shawngr** [comment]
Johnathan

The customer's tenant is a Government (GCC Moderate) tenant.

Thank you,
Shawn Grant |Customer Success Account Manager |Federal
Healthcare and Benefits | Microsoft Corporation
W: (410) 564-2720 | C: (443) 676-8337 | shawngr@microsoft.com<mailto:shawngr@microsoft.com>
US National Support_ 1.800.936.3200 | Non-US National Support_1.800.936.3100

**Comment by FormerlyJelly** [comment]
Hello shawngr,

I have removed some comments / made edits to your replies as they exposed PII for us within Github. Please be careful with responding to email to post your response in GitHub as your personally identifiable information does end up in the body of the issue. Additionally, files attached in email such as screenshots that are meant to be received in Github will not go through.

Thank you for the heads up about the client environment, GCC tenants have a very different security model from a standard client tenant and this could certainly cause issues for your client. Currently we are working to provision Learning Pathways in each tier of GCC security configuration to see if this reproduces under the current version of Learning Pathways. I will respond with findings as soon as possible.

**Comment by shawngr** [comment]
Johnathan,

What direction should direction or status should provide to the customer?

Do you still need know how the Visitors group of the Communications site configured?

Thank you,
Shawn Grant |Customer Success Account Manager |Federal
Healthcare and Benefits | Microsoft Corporation
W: (410) 564-2720 | C: (443) 676-8337 | shawngr@microsoft.com<mailto:shawngr@microsoft.com>
US National Support_ 1.800.936.3200 | Non-US National Support_1.800.936.3100

**Comment by FormerlyJelly** [comment]
Shawn, I have again edited the thread to remove PII from Github.

Please tell the customer that this has reached escalations and is under review. Please also confirm their GCC level when possible! And yes, the security configuration is still an important part of this. If the visitors group is improperly provisioned in regards to access, this may not be an issue that has anything to do with GCC. Thanks much.

**Comment by shawngr** [comment]
The tenant is GCC moderate.  How should I pass along the configuration information you require?

**Comment by FormerlyJelly** [comment]
I really only need to know if people who are targeted by their security configuration - in the Visitors group - have the issues being described. If they do not have access whatsoever in the visitors group, it could explain the behavior.

Could you also pass this along to your client and ask them how they did their installation? Please review when able:
https://techcommunity.microsoft.com/t5/public-sector-blog/using-microsoft-365-learning-pathways-in-your-gcc-high-tenant/ba-p/1932239

If their installation differed from the above, it may also be the cause for the issues seen. This information is not readily available and we will work to add it to the rest of the documentation that is available on GitHub and through Microsoft Docs.

**Comment by shawngr** [comment]
The Visitors group is set to Read, and they added Everyone except External Users as the member.

**Comment by jellis-thrive** [comment]
GCC environment supportability is currently being discussed internally. 
We will provide an update as soon as possible.

**Comment by tcrobbins4** [comment]
So was the final answer that LP and SSS are not working in GCC?  ANything I can do on my end as the global admin?

**Comment by pgdrews** [comment]
I too have a GCC High tenant that provisioning won't even start.  Could be some firewall investigation needed, but based on the above discussion, its likely not worth the effort at this time spend too much troubleshooting time since lower GCC's aren't even supported.

**Comment by tcrobbins4** [comment]
I got it to work in the GCC tenant.  So far everything looks to be working as expected.  Have not started doing any configuration/customization yet, but no more errors.  It had to do with adding the content pak which was hidden under an ellipses "..." menu.    See issue 484.  That's where I got the guidance I needed to get it to work for my particular issue

**Comment by pgdrews** [comment]
@tcrobbins4 interesting.   @jellis-thrive - would it be safe to say GCC High would be a completely different beast?  I would suspect so.  Nothing in GCCH is fun.  So much Commercial remnants exist everywhere making things challenging.

**Comment by pgdrews** [comment]
I guess I'm going to try the manual method outlined in the article above.  https://techcommunity.microsoft.com/t5/public-sector-blog/using-microsoft-365-learning-pathways-in-your-gcc-high-tenant/ba-p/1932239

**Comment by juliemturner** [comment]
Currently we have no way to support GCC or GCCH deployments. I have also seen the Tech Community article but without direct knowledge and access to those types of tenants to do testing we're hard pressed to support issues.  

That all said let's come around to the original post, I have no idea what the real issue is here.
You need to start with the admin page, CustomLearningAdmin.aspx. Is it loading or is it also showing and error? Based on the logs attached, I suspect it's not working. In the original post it looks like the content source for the manifest files were set to an IP address on a non-http endpoint.  That's just not going to work period.  So, if that was the configuration that was done, you're going to need to solve that problem first.  I'm assuming the idea was to bring the manifests behind a firewall so that you wouldn't have to open ports for the github pages url. This is absolutely fine, but you'll need to configure that endpoint for https and makes sure that your root CDN url then has a sub folder for v4... in that folder you need a manifest.json file and then sub folders for each language... 

Long story short based on hearsay and not first-hand knowledge it's my understanding that getting learning pathways running in a GCC moderate tenant is possible, but as I said without access to one myself, I cannot offer any guidance.  I am aware of a community member who is currently embarking on getting LP running in a GCCH tenant... if he manages to do so then I will try and have that person help me document the steps more formally here.

**Comment by juliemturner** [comment]
@shawngr - Please convey my above comment regarding the source of the learning pathways manifest files to the customer and/or have them engage here directly. Thanks.

**Comment by dcashpeterson** [comment]
closed due to inactivity
